README.md

```diff
@@ -146,6 +146,34 @@ WMCO is not responsible for Windows operating system updates. The cluster admini
 creating the VMs and hence, the cluster administrator is responsible for providing an updated image. The cluster 
 administrator can provide an updated image by changing the image in the MachineSet spec.
 
+## Applying autoscaling to WMCO-managed Windows Machines
+
+There are two components to managing resources using autoscaling: a ClusterAutoscaler and MachineAutoscalers.
+The cluster autoscaler is capable of automatically adjusting the size of the cluster based on node resource usage
+and overall deployment needs, within specified constraints. Machine autoscalers modify machine sets
+to scale up and down the number of machines.
+Multiple MachineAutoscalers can be deployed to manage multiple MachineSets.
+Each machine autoscaler object should be mapped to a single machine set.
+
+- Define and deploy a [ClusterAutoscaler](https://docs.openshift.com/container-platform/latest/machine_management/applying-autoscaling.html#configuring-clusterautoscaler).
+- Ensure a MachineSet resource is deployed for Windows machines (see spec in [Usage section](https://github.com/openshift/windows-machine-config-operator#usage)).
+- Create a MachineAutoscaler resource, referencing a MachineSet resource. Replace *\<windows-machineset-name\>*
+  in the template below with the name of Windows MachineSet that was created, then deploy.
```

Comment:
maybe rephrase to specifics -> *name of Windows MachineSet that was created in the previous step, then deploy the MachineAutoscaler resource.